crosswalk/index.html

```diff
@@ -1180,7 +1180,7 @@ <h3><a href="https://www.w3.org/2021/a11y-discov-vocab/latest/#accessibilityFeat
 					<td><a href="https://ns.editeur.org/onix/en/196/24">List 196; Code 24</a>: Dyslexia readability
 					</td>
 					<td>-</td>
-					<td>532 1# $aSpecialised font, character and line spacing, justification and paragraph spacing, and
+					<td>532 1#$aSpecialised font, character and line spacing, justification and paragraph spacing, and
```

Comment:
Same here "532 `<space>` 1#$aSpecialized"
Why no space after the code and the beginning of the description?
I would think it should be 
"532 `<space>` 1#$a `<space>` Specialized"